components/Header.jsx

```diff
@@ -126,14 +126,18 @@ const activeStyle = {
 // TODO: links/routing, mobile
 const Header = () => {
   const classes = useStyles();
-  const [isIconOpen, setIsIconOpen] = React.useState(false);
+  const [isMenuOpen, setIsMenuOpen] = React.useState(false);
```

Comment:
I'm OK with not using a ref (via `React.useRef`), and continuing to use 1 variable to track both whether the menu is open, as well as reference to the IconButton DOM element.

1. I think we ought to just include a comment above this line letting future devs know that this variable contains a dual-purpose. We can say something like:
```jsx
// this variable gets set to the DOM reference of IconButton, non-null value indicates the menu should be visible
const [isMenuOpen, setIsMenuOpen] = React.useState(false);
```

2. I also think it might be worth changing the default value to `null`, rather than `false`. We never call `setIsMenuOpen(true)`, so we may as well be consistent about keeping the domain of this variable to `null` or a DOM node.

```jsx
// this variable gets set to the DOM reference of IconButton, non-null value indicates the menu should be visible
const [isMenuOpen, setIsMenuOpen] = React.useState(null);
```

Reply:
Edits made